website/content/en/highlights/2023-04-11-0-29-0-upgrade-guide.md

```diff
@@ -0,0 +1,31 @@
+---
+date: "2023-04-11"
+title: "0.29 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.29.0"
+authors: ["neuronull"]
+release: "0.29.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.29.0 release includes **deprecations**:
+
+1. [The `datadog` sink has been renamed to `mezmo`](#mezmo_sink)
```

Comment:
Um, the `datadog` sink? 😕

Reply:
accidentally had the file in read only mode so my edits didnt get in 🤦